Shadowlands/Plaguefall/Trash.lua

```diff
@@ -230,6 +237,27 @@ function mod:WitheringFilth(args)
 	self:PlaySound(args.spellId, "alarm")
 end
 
+do
+	local sourceMobId = nil
+
+	local function printTarget(self, name, guid)
+		-- depending on source NPC id it is either CCable or only can be stopped by movement dispellers
+		local movementDispelOnly = sourceMobId == 168907
+		if not movementDispelOnly or self:Dispeller("movement") or self:Healer() or self:Me(guid) then
+			self:TargetMessage(328429, "yellow", name)
+			self:PlaySound(328429, "alert", nil, name)
+			if self:Me(guid) then
+				self:Say(328429)
+			end
+		end
+	end
+
+	function mod:CrushingEmbrace(args)
+		sourceMobId = self:MobId(args.sourceGUID)
+		self:GetUnitTarget(printTarget, 0.5, args.sourceGUID)
```

Comment:
Any better way to pass in sourceMobID? I wish the whole args table was passed through to the callback.